### DISCUSSION TITLE
Caligrafia de Médico

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#522] [Curiosidades]
Post de referência: https://www.vish.blog.br/curiosidades/caligrafia-de-medico/

**Comment by brunoaugusto** [comment]
Uma vez fui ao cardiologista e a receita era praticamente idêntica ao ECG que eu fiz

**Comment by one-eyed-willy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/curiosidades/caligrafia-de-medico/#comment_1087550942>):

> Uma vez fui ao cardiologista e a receita era praticamente idêntica ao ECG que eu fiz

Aposto que o farmacêutico entendeu o que tava escrito

**Comment by brunoaugusto** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/curiosidades/caligrafia-de-medico/#comment_1087554630>):

> Aposto que o farmacêutico entendeu o que tava escrito

Pior que entendeu. Eu peguei o remédio, olhei a quantidade de comprimidos e acabei perguntando se tava certo. A mulher me olhou, deu aquela risadinha simpática e mostrou o que cada rabisco significava.

**Comment by tali-zorah** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/curiosidades/caligrafia-de-medico/#comment_1087557476>):

> Pior que entendeu. Eu peguei o remédio, olhei a quantidade de comprimidos e acabei perguntando se tava certo. A mulher me olhou, deu aquela risadinha simpática e mostrou o que cada rabisco significava.

*Keelah!* :o
